x/dualstaking/keeper/delegate.go

```diff
@@ -359,7 +359,7 @@ func (k Keeper) Redelegate(ctx sdk.Context, delegator, from, to, fromChainID, to
 		)
 	}
 
-	err = k.increaseDelegation(ctx, delegator, to, fromChainID, amount, nextEpoch)
+	err = k.increaseDelegation(ctx, delegator, to, toChainID, amount, nextEpoch)
```

Comment:
I thought this was fixed elsewhere already .. is this PR properly based?

Reply:
i should rebase

Reply:
done